### PR TITLE
Improve tooltip wording of the products settings option

### LINKED ISSUE #13841: Improve tooltip wording of the Products settings' "Default activation status" option
The **"Default activation status"** setting in the Product settings allows to automatically activate/disable products when they are created.

Capture d’écran 2019-05-17 à 16 10 04

But this setting only applies to manual product creation and does not apply to creation through import.

It is not clearly indicated and can lead to confusion for the user.

Capture d’écran 2019-05-17 à 16 10 53

## Specs

**Page:** Shop parameters > Product Settings

**Describe the solution you'd like:**
Modify the tooltip wording of the "Default activation status" setting 

**Current tooltip wording:** "When active, new products will be activated by default during creation."

**New tooltip wording:** "Enable this option if you want to activate by default all your manually created new products."

### PR DESCRIPTION
**Pull request by Mch0**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?           | improve tooltip wording of the products settings option
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #13841
| How to test?      | Juste go in BO page to check the tooltip has been updated

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25462)

### CODE REVIEW

File: translations/default/AdminShopparametersHelp.xlf [hunk 1]
```diff
@@ -701,8 +701,8 @@
         <note>Line: 77</note>
       </trans-unit>
       <trans-unit id="3f141565d26297ff4449250743dba137">
-        <source>When active, new products will be activated by default during creation.</source>
-        <target>When active, new products will be activated by default during creation.</target>
+        <source>Enable this option if you want to activate by default all your manually created new products.</source>
+        <target>Enable this option if you want to activate by default all your manually created new products.</target>
```
Comment (atomiix) [inline translations/default/AdminShopparametersHelp.xlf]: This is not needed, translation files are exported later during the release process ;)

**Review by atomiix — CHANGES_REQUESTED** [review]
Thanks for your contribution @Mch0, just a file to revert and we should be good!

**Review by atomiix — APPROVED** [review]

**Review by Julievrz — APPROVED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
hi @Mch0, thank you for your contribution :)

Seems like the CI is blocking your PR because it contains a merge commit, which is not authorized on the project

**Comment by Mch0** [comment]
Hum seem bad, can i change the branch to start from correct develop branch ? I am going to create a new branch who start from correct develop branch

**Comment by matks** [comment]
> Hum seem bad, can i change the branch to start from correct develop branch ? I am going to create a new branch who start from correct develop branch

This PR is tied to your branch `mcho/#13841-improve-tooltip-wording-of-the-products-settings-option`

If you correct this branch, the PR will automatically update.
If you prefer to create a new branch, you will have to close this PR and open another PR.

**Comment by PierreRambaud** [comment]
Hi @Mch0 ,

Since we had no news from you for more than 30 days, I'll close this pull request. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**